apps/passport-server/src/services/issuanceService.ts

```diff
@@ -963,6 +964,15 @@ export class IssuanceService {
     credential: SemaphoreSignaturePCD
   ): Promise<EdDSATicketPCD[]> {
     return traced("IssuanceService", "issueZuzaluTicketPCDs", async (span) => {
+      const serverUrl = process.env.PASSPORT_CLIENT_URL;
```

Comment:
mentioned offline, but the naming is a bit confusing here. either `imageServerUrl` or `clientUrl` are better

when we rename it here, could we also rename for frogs?